### PR TITLE
Add release notes for 1.19.0

### PR DESCRIPTION
**Pull request by ikhoon**

### CODE REVIEW

**Comment by ikhoon** [comment]
image

File: site/src/pages/release-notes/1.19.0.mdx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+      // Samples all request contexts.
+      return Sampler.always();
+    }
+  }
+  ```
+- <type://DocService> now automatically generates a specification of an annotated service for well known types. #4309 #4322
+  - Java POJO
+  - Kotlin data class
+  - Scala case class
+  - Protocol Buffers' message
```
Comment (ikhoon) [inline site/src/pages/release-notes/1.19.0.mdx:86]:
```suggestion
  - Protocol Buffers' `Message`
```

File: site/src/pages/release-notes/1.19.0.mdx [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+  'ghkim3221',
+  'heowc',
+  'ikhoon',
+  'injae-kim',
+  'j-min5u',
+  'jrhee17',
+  'klurpicolo',
+  'ks-yim',
+  'minwoox',
+  'resquivel-squareup',
```
Comment (jrhee17) [inline site/src/pages/release-notes/1.19.0.mdx:187]: Can you also add [tawAsh1](https://github.com/tawAsh1) for reporting the bug on `ClassCastException`?

Comment (ikhoon) [reply site/src/pages/release-notes/1.19.0.mdx:187]: Thanks for noting it. I missed him.

File: site/src/pages/release-notes/1.19.0.mdx [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+  AggregatedHttpRequest aggregatedReq1 = request.aggregate().join();
+  assert aggregatedReq0 == aggregatedReq1;
+
+  HttpResponse response = ...;
+  AggregatedHttpResponse aggregatedRes0 = response.aggregate().join();
+  AggregatedHttpResponse aggregatedRes1 = response.aggregate().join();
+  assert aggregatedRes0 == aggregatedRes1;
+  ```
+- You can now customize the behavior of auto refresh for `AddressResolverGroup` with
+  <type://DnsResolverGroupBuilder> #4400
```
Comment (minwoox) [inline site/src/pages/release-notes/1.19.0.mdx:22]:
```suggestion
  <type://DnsResolverGroupBuilder>. #4400
```

File: site/src/pages/release-notes/1.19.0.mdx [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+    }
+  }
+  ```
+- <type://DocService> now automatically generates a specification of an annotated service for well known types. #4309 #4322
+  - Java POJO
+  - Kotlin data class
+  - Scala case class
+  - Protocol Buffers' `Message`
+  - ScalaPB's `GeneratedMessage`
+- You can now upload files to your <type://GraphqlService> when using
```
Comment (minwoox) [inline site/src/pages/release-notes/1.19.0.mdx:88]:
```suggestion
- You can now upload files to your <type://GraphqlService> using 
```

File: site/src/pages/release-notes/1.19.0.mdx [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+  ```
+- <type://DocService> now automatically generates a specification of an annotated service for well known types. #4309 #4322
+  - Java POJO
+  - Kotlin data class
+  - Scala case class
+  - Protocol Buffers' `Message`
+  - ScalaPB's `GeneratedMessage`
+- You can now upload files to your <type://GraphqlService> when using 
+  [GraphQL multipart requests](https://github.com/jaydenseric/graphql-multipart-request-spec). #3971 #4199
+- You can now use `armeria-version-catalog` module to easily import Armeria dependencies
```
Comment (minwoox) [inline site/src/pages/release-notes/1.19.0.mdx:90]:
```suggestion
- You can now use `armeria-version-catalog` module to easily import Armeria and Armeria's direct dependencies
```

File: site/src/pages/release-notes/1.19.0.mdx [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+      create("armeriaDeps") {
+        from("com.linecorp.armeria:armeria-version-catalog:1.19.0")
+      }
+    }
+  }
+  ```
+
+## 📈 Improvements
+
+- `CompositeException` now shows full stack traces for all nested exceptions
```
Comment (minwoox) [inline site/src/pages/release-notes/1.19.0.mdx:105]:
```suggestion
- `CompositeException` now shows full stack traces of all nested exceptions 
```

File: site/src/pages/release-notes/1.19.0.mdx [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+  @ProducesJson
+  public Publisher<Void> noBody() {
+    return Mono.empty();
+  }
+  ```
+- Armeria server no longer sends chunked encoding header with `304 Not Modified` status. #4386
+- HTTP/2 graceful shutdown timeout is correctly set using <type://ClientFactory#idleTimeoutMillis()>. #4381
+- Unframed <typeplural://GrpcService> now support `application/x-protobuf` and `application/x-google-protobuf`
+  media types. #4355 #4364
+- You can now use a custom `JsonFormat.Printer` for <type://ProtobufResponseConverterFunction>  #4301 #4336
```
Comment (minwoox) [inline site/src/pages/release-notes/1.19.0.mdx:134]:
```suggestion
- You can now use a custom `JsonFormat.Printer` for <type://ProtobufResponseConverterFunction>.  #4301 #4336
```

File: site/src/pages/release-notes/1.19.0.mdx [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+
+## 📃 Documentation
+
+- You can now learn how to write a gRPC service and client with Armeria by walking through
+   [the gRPC tutorial](/tutorials/grpc/blog). #3987
```
Comment (minwoox) [inline site/src/pages/release-notes/1.19.0.mdx:139]:
```suggestion
- You can now learn how to write a gRPC service and client with Armeria by walking through
  [the gRPC tutorial](/tutorials/grpc/blog). #3987
```

**Review by minwoox — APPROVED** [review]
Thanks, @ikhoon ❤️

File: site/src/pages/release-notes/1.19.0.mdx [hunk 9]
```diff
@@ -0,0 +1,189 @@
+---
+date: 2022-09-08
+---
+
+## 🌟 New features
+
+- You can now repeatedly aggregate an <type://HttpRequest> and <type://HttpResponse> when using
+  <type://HttpRequest#aggregate()> and <type://HttpResponse#aggregate()>. If an <type://HttpRequest> and
+  <type://HttpResponse> is aggregated before, the cached value is returned for the next calls. #3789 #4366
```
[the remark this answers is not in the document]
Comment (ikhoon) [reply site/src/pages/release-notes/1.19.0.mdx:9]: `AggregatedHttpMessage` is not a public API. Let me list them all.

File: site/src/pages/release-notes/1.19.0.mdx (hunk 7 above)
Comment (ikhoon) [reply site/src/pages/release-notes/1.19.0.mdx:134]: I was confused about this. 😅
<type://...?full> does not work if `.` is next to it.

File: site/src/pages/release-notes/1.19.0.mdx (hunk 9 above)
[the remark this answers is not in the document]
Comment (ikhoon) [reply site/src/pages/release-notes/1.19.0.mdx]: Sorry. It became a public API recently by my work. Anyway, `AggregatedHttpRequest` and `AggregatedHttpResponse` look clear. I will pick this version.

File: site/src/pages/release-notes/1.19.0.mdx [hunk 10]
```diff
@@ -0,0 +1,190 @@
+---
+date: 2022-09-08
```
Comment (ikhoon) [inline site/src/pages/release-notes/1.19.0.mdx:2]:
```suggestion
date: 2022-09-13
```

**Pull request merged**